pkg/util/mount/mount.go

```diff
@@ -143,6 +142,23 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, devPath string) ([]string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return nil, err
+	}
+	// Find all references to the device.
+	var refs []string
+	for i := range mps {
+		if mps[i].Path == devPath || mps[i].Device == devPath {
```

Comment:
So in the example you gave above, we do not want to count that mount point because it is the global mount and not the pod mount.  We only want to count mount points that are the bind mounts into the pods.